services/utils.js

```diff
@@ -0,0 +1,14 @@
+function sanitize(object) {
+    var result = JSON.parse(JSON.stringify(object)); // tricky but efficient way to clone an object
```

Comment:
I updated to the latest version of node (as of ~3 months ago) so I think you can just use Object.assign({}, object). I think it's faster or roughly equivalent in terms of speed.